docs/quickstart.rst

```diff
@@ -34,22 +34,21 @@ Trying Some Examples
 --------------------
 
 The Scenic repository contains many example scenarios, found in the :file:`examples` directory.
-They are organized by the simulator they are written for, e.g. :abbr:`GTA (Grand Theft Auto V)` or Webots; there are also cross-platform scenarios written for Scenic's abstract application domains, e.g. the :ref:`driving domain <driving_domain>`.
-
+They are organized in various directories with the name of the simulator, abstract application domain, or visualizer they are written for. For example, ``gta`` and ``webots`` for the :abbr:`GTA (Grand Theft Auto V)` and Webots simulators; the `driving` directory for the abstract :ref:`driving domain <driving_domain>`; and the ``visualizer`` directory for the built in Scenic visualizer.
 
 Each simulator has a specialized Scenic interface which requires additional setup (see :ref:`simulators`); however, for convenience Scenic provides an easy way to visualize scenarios without running a simulator.
 Simply run :command:`scenic`, giving a path to a Scenic file:
 
 .. code-block:: console
 
-	$ scenic examples/webots/vacuum/vacuum_simple.scenic
+	$ scenic --axes examples/webots/vacuum/vacuum_simple.scenic
```

Comment:
I'd just remove the axes option, so that the description in the previous line is accurate; the axes aren't informative here anyway (the viewpoint is the same as for the 2D screenshots, which don't show the axes). We can describe the axes option in the tutorial instead (you can just move the parenthetical you have about it below).